packages/augur-ui/src/modules/portfolio/containers/my-markets.ts

```diff
@@ -23,7 +25,7 @@ const mapStateToProps = (state) => {
 // TOJDO confirm with TOm whats up with this, getBalance Only
 const mapDispatchToProps = (dispatch) => ({
   removePendingMarket: (id) => dispatch(removePendingData(id, CREATE_MARKET)),
-  createMarketRetry: (data) => dispatch(createMarketRetry(data)),
+  createMarketRetry: (data) => dispatch(submitNewMarket(null, data, true)),
```

Comment:
here is a place to have a more descriptive method call, maybe call it `retryCreateMarket` and only pass in `data`